src/test/unit/profile-data.test.js

```diff
@@ -682,239 +665,6 @@ describe('symbolication', function() {
   // TODO: check that functions are collapsed correctly
 });
 
-describe('upgrades', function() {
```

Comment:
I broke these out into their own file, since they were really hard to find, and organized in an inconsistent fashion.